notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js

```diff
@@ -25,148 +25,148 @@ var upnotice = upnotice || {};
 
 if (!upnotice.init) {
 
-  upnotice.init = true;
-
-  (function() {
-
-    var defaults = {
-      selectors: {
-        template:        '.template',
-        title:           '.title',
-        body:            '.body',
-        link:            '.link',
-        actions:         '.notification-actions',
-        actionTemplate:  '.action-template',
-        summaryTemplate: '.summary-template'
-      },
-      readyCallback: function() {}
-    };
-
-    // First 'prime-the-pump' with an ActionURL
-    function initNotices($, settings, callback) {
-      $.ajax({
-        type: 'POST',
-        url: settings.invokeNotificationServiceUrl,
-        async: false,
-        success: function() {
-          fetchNotices($, settings, callback);
-        }
-      });
-    }
-
-    // Then fetch the notifications with a ResourceURL
-    function fetchNotices($, settings, callback) {
-      $.ajax({
-        url      : settings.getNotificationsUrl,
-        type     : 'POST',
-        dataType : 'json',
-        success: function (data) {
-          feed = data.feed;
-          callback(feed);
-        },
-        error: function () {
-          container.html(" ").text("AJAX failed. ~ THE END ~");
+    upnotice.init = true;
+
+    (function() {
+
+        var defaults = {
+            selectors: {
+                template:        '.template',
+                title:           '.title',
+                body:            '.body',
+                link:            '.link',
+                actions:         '.notification-actions',
+                actionTemplate:  '.action-template',
+                summaryTemplate: '.summary-template'
+            },
+            readyCallback: function() {}
+        };
+
+        // First 'prime-the-pump' with an ActionURL
+        function initNotices($, settings, callback) {
+            $.ajax({
+                type: 'POST',
+                url: settings.invokeNotificationServiceUrl,
+                async: false,
+                success: function() {
+                    fetchNotices($, settings, callback);
+                }
+            });
         }
-      });
 
-    }
+        // Then fetch the notifications with a ResourceURL
+        function fetchNotices($, settings, callback) {
+            $.ajax({
+                url      : settings.getNotificationsUrl,
+                type     : 'POST',
+                dataType : 'json',
+                success: function (data) {
+                    var feed = data.feed;
+                    callback(feed);
+                },
+                error: function () {
+                    container.html(' ').text('AJAX failed. ~ THE END ~');
```

Comment:
Heh.  It's a pretty poor error message too.  Most end users won't have a clue to what that means.